api/k8s/v1/k8s.proto

```diff
@@ -1055,6 +1063,53 @@ message DescribeNamespaceResponse {
   Namespace namespace = 1 [ (clutch.api.v1.log) = false ];
 }
 
+// ObjectKind represent a persistent entity in the system
+// For now we only support the kind Pod, we can expand on this
+// list as we add support for different kinds.
+enum ObjectKind {
+  UNSPECIFIED = 0;
+  UNKNOWN = 1;
+  POD = 2;
+}
+
+// this represents a Kubernetes event: https://pkg.go.dev/k8s.io/api/core/v1#Event
+// for a given object
+message Event {
+  string name = 1;
+  string reason = 2;
+  string description = 3;
```

Comment:
nit, to match the upstream k8s event object
```suggestion
  string message = 3;
```

Reply:
The reason I called it `description` instead of `message` was I thought event.Description would be more clear than event.Message but if we want to stick to the upstream naming that works too